lib/la_maquina/dependency_map/yaml_map.rb

```diff
@@ -5,6 +5,7 @@ def find(*args)
         submap = map
         args.each do |key|
           submap = submap[key]
+          break if submap.blank?
```

Comment:
What do you think about throwing a `LaMaquina::Errors::InvalidMappingError`? Is it better for the user to know that there's an unexpected mapping and handle it, rather than the map quietly swallowing errors?

And then the Piston can catch this error and notify it to something non-exceptiony/eat it. I am not convinced that quietly eating a map miss is the best way to handle this. 

What do you think?